### PR TITLE
Add some more commands

### PR DESCRIPTION
**Pull request by Laarryy**
!ssl - for ssl error
!extracontexts - to link to wiki PR for extracontexts
!hosting - permanent, auto updating link to latest host discussion on the admincraft reddit (courtesy of Turbo)
!inheritance - link to wiki usage-inheritance page

### CODE REVIEW

**Comment by Laarryy** [comment]
Hope that extracontexts, linking to the PR for the wiki, is fine!

**Comment by Turbotailz** [comment]
Should be fine as long as @lucko merges that before this.

File: modules/commands/list.json [hunk 1]
```diff
@@ -296,6 +306,14 @@
     "description": "Learn what each of the config options are for.",
     "wiki": true
   },
+  {
+    "name": "extracontexts",
+    "aliases": "extracontext",
+    "title": "Extra Contexts",
+    "url": "https://github.com/lucko/LuckPerms/wiki/Context#ExtraContexts",
```
Comment (lucko) [inline modules/commands/list.json:313]: We can just link to https://github.com/LuckPerms/ExtraContexts
```suggestion
    "url": "https://github.com/LuckPerms/ExtraContexts",
```

File: modules/commands/list.json [hunk 2]
```diff
@@ -345,7 +363,7 @@
     "fields": [
       {
         "key": "Additional downloads",
-        "value": "https://ci.lucko.me/view/LuckPerms/"
+        "value": "https://ci.lucko.me/job/LuckPerms/"
```
Comment (lucko) [inline modules/commands/list.json:366]: Linking to `view/` is correct here I think. luckperms.net already has the main downloads, "additional" should include everything else available on Jenkins which is LP related.

File: modules/commands/list.json [hunk 3]
```diff
@@ -500,6 +518,56 @@
...
+    "title": "The useSSL error!",
+    "description": "The useSSL error appears when you have not added the correct lines in the right place in the configuration. Make sure you do not uncomment the ones already there, but rather add them underneath the `characterEncoding` line.",
+    "fields": [
+      {
+        "key": "Read me!",
+        "value": "https://github.com/lucko/LuckPerms/wiki/Storage-system-errors#mysql-ssl-errors"
+      },
+      {
+        "key": "What is SSL?",
+        "value": "https://www.cloudflare.com/learning/ssl/what-is-ssl/"
```
Comment (lucko) [inline modules/commands/list.json:535]: Don't think it's necessary to link this.

Plus it talks about SSL mainly in the context of HTTPS, which is not relevant here.

File: modules/commands/list.json [hunk 4]
```diff
@@ -500,6 +518,56 @@
...
+      }
+    ]
+  },
+  {
+    "name": "timeout",
+    "aliases": [
+      "connectiontimeout"
+    ],
+    "title": "The connectionTimeout error!",
+    "description": "The useSSL error appears when you have not added the correct lines in the right place in the configuration. Make sure you do not uncomment the ones already there, but rather add them underneath the `characterEncoding` line.",
```
Comment (lucko) [inline modules/commands/list.json:545]: How is this any different from the command which links to the storage errors wiki page?

Don't see why we need to double up.

File: modules/commands/list.json [hunk 5]
```diff
@@ -500,6 +518,56 @@
...
+        "value": "https://github.com/lucko/LuckPerms/wiki/Storage-system-errors#mysql-ssl-errors"
+      },
+      {
+        "key": "What is SSL?",
+        "value": "https://www.cloudflare.com/learning/ssl/what-is-ssl/"
+      }
+    ]
+  },
+  {
+    "name": "hosting",
```
Comment (lucko) [inline modules/commands/list.json:558]: Sorry, I think this is out of scope for a bot command.

Comment (Laarryy) [reply modules/commands/list.json:558]: This was an artefact I thought I had removed, haha. Will fix when I get back to the PC

File: modules/commands/list.json (hunk 4 above)
Comment (Laarryy) [reply modules/commands/list.json:545]: This is mainly just for convenience, in that we can link straight to the really common error and that's about it 🤷‍♂️ 

I'll remove it when I can if you don't tell me not to

Comment (Andre601) [reply modules/commands/list.json:545]: Also, why does it say `The connectionTimeout error!` (Why even connectionTimeout and not just Connection Timeout?) when it is about the SSL error?
Did you just forget to update the links and fields here? because it looks like it.

Comment (Laarryy) [reply modules/commands/list.json:545]: Yes, I did, it's a leftover, sorry.

File: modules/commands/list.json (hunk 3 above)
Comment (Laarryy) [reply modules/commands/list.json]: removed

File: modules/commands/list.json (hunk 4 above)
Comment (Laarryy) [reply modules/commands/list.json]: removed

File: modules/commands/list.json (hunk 5 above)
Comment (Laarryy) [reply modules/commands/list.json]: removed

**Comment by Laarryy** [comment]
@lucko I've removed em all, just extracontexts now

**Comment by lucko** [comment]
For future reference, you can look at the "Files changed" tab in GitHub to see a summary of your PRs changes - super easy to spot mistakes in that view! ;)
